Navaids/FIXES_UK.txt

```diff
@@ -587,6 +587,7 @@ NINEB N054.03.44.000 W004.07.42.000
 NINEX N058.51.20.000 W009.30.00.000
 NIPIT N054.27.09.000 W008.24.10.000
 NISBI N053.21.29.000 W000.39.21.000
+NISKA N054.15.00.000 W002.30.33.000
 NITON N052.33.12.000 W003.12.01.000
```

Comment:
Will NISKA (and VAPPI) sit on an airway or are they for this procedure only? Are they defined in ENR4.4? If not, maybe it's better to go in /EGPF/Fixes.txt

Reply:
They're both defined in ENR 4.4 and are noted as "EGPF/EGPK STARs" so it would seem to me that it's an appropriate place https://i.imgur.com/IrfuUMQ.png